src/troute-routing/troute/routing/fast_reach/mc_reach.pyx

```diff
@@ -222,8 +222,8 @@ cpdef object compute_network_structured(
     # Check shapes
     if qlat_values.shape[0] != data_idx.shape[0]:
         raise ValueError(f"Number of rows in Qlat is incorrect: expected ({data_idx.shape[0]}), got ({qlat_values.shape[0]})")
-    if qlat_values.shape[1] > nsteps:
-        raise ValueError(f"Number of columns (timesteps) in Qlat is incorrect: expected at most ({data_idx.shape[0]}), got ({qlat_values.shape[0]}). The number of columns in Qlat must be equal to or less than the number of routing timesteps")
+    if qlat_values.shape[1] < nsteps:
```

Comment:
Is this change from "greater than" to "less than" intentional? Not sure how this relates to the HYFeatures changes

Reply:
When running the included routing test I noticed this bug, and fixed it so that a simple one time step routing test could be performed.